numba/tests/test_parfors.py

```diff
@@ -525,13 +634,37 @@ def test_impl(A):
         self.check(test_impl, A)
         A = np.random.randint(10, size=n).astype(np.int32)
         self.check(test_impl, A)
+
         # test checking the number of arguments for the reduce function
         def test_impl():
             g = lambda x: x ** 2
             return reduce(g, np.array([1, 2, 3, 4, 5]), 2)
         with self.assertTypingError():
             self.check(test_impl)
 
+        # test checking reduction over bitarray masked arrays
+        n = 160
+        A = np.random.randint(10, size=n).astype(np.int32)
+        def test_impl(A):
+            return np.sum(A[A>=3])
+        self.check(test_impl, A)
+        # TODO: this should fuse
```

Comment:
Are the TODOs here going to be done in this PR or to act as reminders?

Reply:
These are reminders. There is more core work to be done before masked reductions work properly.